### PR TITLE
fixed mandrill email sends

### PR DESCRIPTION
**Pull request by mqueme**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4381
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Mandrill was not sending emails.
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to your configuration settings, and click on Email settings
2. Select Mandrill as your service to send emails through
3. Test connection and send a test email
4. Save and try sending an email to any of your contacts
4. Email sends would fail.

#### Steps to test this PR:
1. Apply this PR and follow steps above
2. Emails should now send

### CODE REVIEW

File: app/bundles/EmailBundle/Config/config.php [hunk 1]
```diff
@@ -307,7 +307,7 @@
                 'serviceAlias' => 'swiftmailer.mailer.transport.%s',
                 'methodCalls'  => [
                     'setUsername'      => ['%mautic.mailer_user%'],
-                    'setPassword'      => ['%mautic.mailer_password%'],
+                    'setPassword'      => ['%mautic.mailer_api_key%'],
```
Comment (alanhartless) [inline app/bundles/EmailBundle/Config/config.php]: Is this a BC break?

**Comment by mqueme** [comment]
I don't think so, it's the internal name we use for the parameter.

**Comment by mqueme** [comment]
This was the bug itself. On the form it's called api_key

File: app/bundles/EmailBundle/Config/config.php (hunk 1 above)
Comment (escopecz) [inline app/bundles/EmailBundle/Config/config.php]: It doesn't seem like it: https://github.com/mautic/mautic/blob/882248a3707a173e58f6a475bdc6c93410b575db/app/bundles/EmailBundle/Form/Type/ConfigType.php#L443

**Pull request merged**